types/components/useAccordionToggle.d.ts

```diff
@@ -0,0 +1,5 @@
+export default interface useAccordionToggle {
```

Comment:
Maybe it'd make the most sense to just make this a named export from `./AccordionToggle`? And in `src/` as well.

Reply:
hmm, it would be a breaking change if we changed the export in `src/`, no?

Reply:
yeah, but we're still in betas :p